main.js

```diff
@@ -149,6 +150,7 @@ function createJitsiMeetWindow() {
     initPopupsConfigurationMain(mainWindow);
     setupAlwaysOnTopMain(mainWindow);
     setupPowerMonitorMain(mainWindow);
+    setupScreenSharingMain(mainWindow, 'Jitsi Meet');
```

Comment:
Please don't hardcode this. Use https://github.com/jitsi/jitsi-meet-electron/blob/3c448837e771796018dfff46dda2b89c7e5f98e2/app/features/config/index.js#L11

Reply:
ok, will create a PR to fix this.

Reply:
Thanks!